plugins/service_discovery/static.md

```diff
@@ -0,0 +1,92 @@
+# Static Service Discovery Plugin
+
+The `static` service discovery plugin set the list of targets.
+
+## Example Configuration
+
+This is the exmple with out_forward.
+It define the list of targets. it is similer to the server directive in out_forward.
+
+```
+<source>
+  @type forward
+
+  <service_discovery>
+    @type static
+    host 127.0.0.1
```

Comment:
should wrap `<service>`?

Reply:
You mean as below?

```
<service_discovery>
   @type static
   <server>
     host 127.0.0.1
   </server>
</service_discovery>
```
I don't think so. see https://github.com/fluent/fluentd/pull/2541#issue-304197753 description.